src/Panel/Plugins.php

```diff
@@ -92,6 +112,16 @@ public function read(string $type): string
 			}
 		}
 
+		if ($type === 'mjs') {
+			if (empty($dist)) {
+				return '';
+			}
+
+			$modules = Json::encode($dist);
+			$modulePromise = "Promise.all($modules.map(url => import(url)))";
+			return "try { await $modulePromise } catch (e) { console.error(e) }" . PHP_EOL;
```

Comment:
Why EOL specifically? Given that we use `\n` throughout the code base, shouldn't we also use it here?